governance/votes/Executive vote - June 18, 2021.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Template - [Executive Vote] Flash Mint Implementation, Debt Floor Changes - June 18, 2021
+
+---
+# [Executive Proposal] Flash Mint Implementation, Debt Floor Changes - June 18, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- The [Flash Mint Module](https://mips.makerdao.com/mips/details/MIP25) will be activated and authorized to flash-mint DAI.
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The [Debt Floor](https://community-development.makerdao.com/en/learn/governance/param-debt-floor) (`dust`) parameter for all vault types *excluding* PSM-USDC-A, ETH-B and ETH-C will be increased from 5,000 DAI to 10,000 DAI.
+- The [Debt Floor](https://community-development.makerdao.com/en/learn/governance/param-debt-floor) (`dust`) parameter for the ETH-B vault type will be increased from 15,000 DAI to 30,000 DAI.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal **does not** include an office-hours modifier that would require it to be cast between the hours of 14:00 and 21:00 UTC.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Flash Mint Module Activation
+
+As per the outcome of the November 2020 Monthly Governance Cycle [executive](https://vote.makerdao.com/executive/5fbb7ab169fd97001ae7442b?network=mainnet#proposal-detail). The Flash Mint Module has been implemented and deployed by the Protocol Engineering Core Unit. The Flash Mint Module has the following governable parameters:
+
+`Max`: Denotes the maximum amount of DAI that can be flash-minted at any one time.  
```

Comment:
I think this is fine, but given Sam's explanation yesterday, I wonder if this needs to be clarified a little. Don't have a great way of saying it though, maybe:

`Max`: Denotes the maximum increment of DAI that can be minted per interaction with the contract.